### PR TITLE
Upm support

### PR DESCRIPTION
**Pull request by bilal-arikan**
Hi,
I moved the plugins folders,
created package.json,
removed default settings asset

I haven't tried it in a real project yet.

### CODE REVIEW

File: Assets/GameJolt/Demo.meta [hunk 1]
```diff
@@ -1,9 +1,8 @@
 fileFormatVersion: 2
-guid: b91b6ee24b8365c46bc58bc634563d1b
+guid: 001272e13cb1abe428a1e460e727d0fc
```
Comment (InfectedBytes) [inline Assets/GameJolt/Demo.meta]:
```suggestion
guid: b91b6ee24b8365c46bc58bc634563d1b
```
don't change any guids of existing files

File: Assets/GameJolt/Runtime.meta [hunk 2]
```diff
@@ -1,9 +1,8 @@
 fileFormatVersion: 2
-guid: 9b696b31cf14e0542ba0f0d864c955dd
+guid: ab05197c209c5c7488f63c852f3d61f7
```
Comment (InfectedBytes) [inline Assets/GameJolt/Runtime.meta]:
```suggestion
guid: 9b696b31cf14e0542ba0f0d864c955dd
```
don't modify the guids of existing files

File: README.md [hunk 3]
```diff
@@ -3,6 +3,11 @@ Game Jolt [Game API](http://gamejolt.com/api/doc/game/) wrapper for [Unity](http
 
 Check out the [documentation](http://InfectedBytes.github.io/gj-unity-api/).
 
+## How to use with Unity Package Manager
+Add to manifest.json
+```
+"com.gamejolt.api": "https://github.com/bilal-arikan/gamejolt-unity-api.git?path=/Assets/GameJolt#2.5.7",
```
Comment (InfectedBytes) [inline README.md]: please use the main repository https://github.com/InfectedBytes/gj-unity-api.git?path=/Assets/GameJolt#2.5.7
```suggestion
"com.gamejolt.api": "https://github.com/InfectedBytes/gj-unity-api.git?path=/Assets/GameJolt#2.5.7",
```

**Review by InfectedBytes — CHANGES_REQUESTED** [review]
Sorry for the late response, but I finally got some time to check your PR out.

**Comment by bilal-arikan** [comment]
Thanks for your response,

Changing value of `Editor\GameJolt\Tools.cs 'ManagerPrefabPath'`  probably broke the old .unitypackage file structure. It may be necessary to find a solution to this before merge.

**Comment by InfectedBytes** [comment]
> Thanks for your response,
> 
> Changing value of `Editor\GameJolt\Tools.cs 'ManagerPrefabPath'` probably broke the old .unitypackage file structure. It may be necessary to find a solution to this before merge.

right, good point. Probably best to keep both paths, if the old path exists -> move file (+meta file) to new path

**Comment by InfectedBytes** [comment]
I've created a upm-support branch in my repo too, so if you just retarget this PR to that branch, we can merge it and I'll take care of some open issues, like upgrading from older packages and stuff.

**Pull request closed without merging**